### PR TITLE
fix: unify .wundergraph directory resolution

### PR DESCRIPTION
**Pull request by StarpTech**
#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: cli/commands/generate.go [hunk 1]
```diff
@@ -27,9 +27,21 @@ var generateCmd = &cobra.Command{
...
+		}
+
+		// only validate if the file exists
+		_, err = files.GetWunderGraphConfigFilePath(wgDir, configEntryPointFilename)
+		if err != nil {
+			return err
+		}
+
+		// optional
+		codeServerFilePath, _ := files.GetWunderGraphServerFilePath(wgDir, configEntryPointFilename)
```
Comment (spetrunin) [inline cli/commands/generate.go:42]: I believe it should use `serverEntryPointFilename`

File: cli/commands/generate.go [hunk 2]
```diff
@@ -138,8 +150,8 @@ Use this command if you only want to generate the configuration`,
 
 		configBundler := bundler.NewBundler(bundler.Config{
 			Name:          "config-bundler",
-			AbsWorkingDir: entryPoints.WunderGraphDirAbs,
-			EntryPoints:   []string{configEntryPointFilename},
+			AbsWorkingDir: wgDir,
+			EntryPoints:   []string{serverEntryPointFilename},
```
Comment (spetrunin) [inline cli/commands/generate.go:154]: is it correct that here we use server entryPoint? previously it was a config

File: pkg/files/files.go [hunk 3]
```diff
@@ -24,56 +24,58 @@ func FileExists(filePath string) bool {
...
-	} else {
-		files.WunderGraphDirAbs = wundergraphAbsDir
+		return dir, nil
 	}
 
-	configEntryPoint := path.Join(files.WunderGraphDirAbs, configEntryPointFilename)
+	return absWgDir, nil
+}
+
+// GetWunderGraphConfigFilePath returns the absolute path to the wundergraph.config.ts file.
```
Comment (spetrunin) [inline pkg/files/files.go:61]: These 2 methods look the same, probably we could use only 1 unified
cause they produce errors which differed only in filename from param

File: cli/commands/up.go [hunk 4]
```diff
@@ -35,9 +35,21 @@ var upCmd = &cobra.Command{
...
+		}
+
+		// only validate if the file exists
+		_, err = files.GetWunderGraphConfigFilePath(wgDir, configEntryPointFilename)
+		if err != nil {
+			return err
+		}
+
+		// optional
+		codeServerFilePath, _ := files.GetWunderGraphServerFilePath(wgDir, configEntryPointFilename)
```
Comment (spetrunin) [inline cli/commands/up.go:50]: should be server also?

**Review by spetrunin — CHANGES_REQUESTED** [review]

File: cli/commands/generate.go (hunk 1 above)
Comment (StarpTech) [reply cli/commands/generate.go:42]: I got rid of the `EntryPoint` naming schema.

File: cli/commands/generate.go (hunk 2 above)
Comment (StarpTech) [reply cli/commands/generate.go:154]: Good catch!

File: pkg/files/files.go (hunk 3 above)
Comment (StarpTech) [reply pkg/files/files.go]: Done.

File: cli/commands/generate.go [hunk 5]
```diff
@@ -27,9 +27,21 @@ var generateCmd = &cobra.Command{
...
+
+		// only validate if the file exists
+		_, err = files.GetValidFilePath(wgDir, configEntryPointFilename)
+		if err != nil {
+			return fmt.Errorf(`code file "%s" not found`, configEntryPointFilename)
+		}
+
+		// optional
+		codeServerFilePath, _ := files.GetValidFilePath(wgDir, serverEntryPointFilename)
+		if err != nil {
```
Comment (spetrunin) [inline cli/commands/generate.go:43]: which error are we checking here?

File: cli/commands/generate.go [hunk 6]
```diff
@@ -27,9 +27,21 @@ var generateCmd = &cobra.Command{
...
+			return fmt.Errorf(`code file "%s" not found`, serverEntryPointFilename)
```
Comment (spetrunin) [inline cli/commands/generate.go:44]: do we need to repeat the error shape from the helper?

Comment (StarpTech) [reply cli/commands/generate.go:44]: What do you mean?

**Review by spetrunin — APPROVED** [review]

**Pull request merged**